raresGabriel01/home-assignment
# [ISSUE-4] Storybook setup

# Issue link
#4 
# Description
A basic storybook setup. Generated a dummy `t-grid` component using `ng generate` for testing purposes.
# How to test?
1. open a terminal
2. `cd` into the root of the project
3. run `npm run storybook`
4. the browser should open the storybook
5. you should see a dummy `t-grid` option on the left-side menu

# Attachments
![image](https://github.com/raresGabriel01/home-assignment/assets/56756497/e33833af-20c4-4390-8ff8-98da2534db60)
